contrib/completion/bash/docker

```diff
@@ -104,6 +104,22 @@ __docker_pos_first_nonflag() {
 	echo $counter
 }
 
+# Transforms a multiline list of strings into a single line string
+# with the words separated by "|".
+# This is used to prepare arguments to __docker_pos_first_nonflag().
+__docker_to_alternatives() {
+	local parts=( $1 )
+	local IFS='|'
```

Comment:
Ah `local`, I always forget about `local`.  This is much better than using `unset` for such a small function. :heart: